scripts/cluster/join.py

```diff
@@ -730,6 +752,10 @@ def join_dqlite(connection_parts):
     master_ep = connection_parts[0].split(":")
     master_ip = master_ep[0]
     master_port = master_ep[1]
+
+    if len(connection_parts) > 2 and verify:
```

Comment:
Spitting on `:` to select IP and port in this fashion probably prevents IPv6 addresses from being used.

Reply:
You are right, we have not looked into IPv6 support yet. Supporting IPv6 will probably need to touch other places as well. I am making a note to revisit this part when we start working on IPv6.

Reply:
I'd recommend using `urlparse` regardless.

Reply:
Because this is not a url urlparse will not parse it correctly.
```
>>> urlparse('127.0.0.1:2500')
ParseResult(scheme='', netloc='', path='127.0.0.1:2500', params='', query='', fragment='')
```